src/Models/Device.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace StreamChat.Models
 {
-    public static class PushProvider
+    public static class PushProviderNames
```

Comment:
in strict sense, it should be `Types` and also it's a breaking change

Reply:
technically yes but I don't think anyone uses this :D

Reply:
most likely